libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/TelemetryUserFeedback.kt

```diff
@@ -25,8 +25,7 @@ class TelemetryUserFeedback(
     val locationsAfter: Array<Location>,
     val feedbackId: String,
     val screenshot: String? = null,
-    val step: TelemetryStep? = null,
-    var metadata: TelemetryMetadata
```

Comment:
Could we remove `TelemetryUserFeedback` completely (and therefore `TelemetryStep`) and rely only on `NavigationFeedbackEvent`? It sounds like they're duplicated unnecessarily.

Reply:
👍